Gemfile.lock

```diff
@@ -0,0 +1,347 @@
+    ipaddress (0.8.3)
+    jmespath (1.3.1)
+    json (1.8.6)
+    jwt (1.5.6)
+    mail (2.6.6)
+      mime-types (>= 1.16, < 4)
+    memoizable (0.4.2)
+      thread_safe (~> 0.3, >= 0.3.1)
+    metaclass (0.0.1)
+    method_source (0.8.2)
+    mime-types (2.99.3)
+    mini_portile2 (2.3.0)
+    mocha (0.14.0)
+      metaclass (~> 0.0.1)
+    multi_json (1.12.2)
+    multi_xml (0.6.0)
+    multipart-post (2.0.0)
+    naught (1.1.0)
+    net-scp (1.2.1)
+      net-ssh (>= 2.6.5)
+    net-sftp (2.1.2)
+      net-ssh (>= 2.6.5)
+    net-ssh (3.2.0)
+    netrc (0.11.0)
+    nokogiri (1.8.1)
+      mini_portile2 (~> 2.3.0)
+    oauth2 (1.4.0)
+      faraday (>= 0.8, < 0.13)
+      jwt (~> 1.0)
+      multi_json (~> 1.3)
+      multi_xml (~> 0.5)
+      rack (>= 1.2, < 3)
+    open4 (1.3.0)
+    pagerduty (2.0.0)
+      json (>= 1.7.7)
+    parser (2.4.0.0)
+      ast (~> 2.2)
+    powerpack (0.1.1)
+    pry (0.10.2)
+      coderay (~> 1.1.0)
+      method_source (~> 0.8.1)
+      slop (~> 3.4)
+    public_suffix (3.0.0)
+    qiniu (6.8.1)
+      json (~> 1.8)
+      mime-types (~> 2.4, >= 2.4.0)
+      rest-client (~> 2.0, >= 2.0.0)
+      ruby-hmac (~> 0.4)
+    rack (2.0.3)
+    rainbow (2.2.2)
+      rake
+    rake (12.1.0)
+    rb-fsevent (0.9.4)
+    rb-inotify (0.9.3)
+      ffi (>= 0.5.0)
+    rbvmomi (1.11.3)
+      builder (~> 3.0)
+      json (>= 1.8)
+      nokogiri (~> 1.5)
+      trollop (~> 2.1)
+    redcarpet (3.0.0)
+    rest-client (2.0.2)
+      http-cookie (>= 1.0.2, < 2.0)
+      mime-types (>= 1.16, < 4.0)
+      netrc (~> 0.8)
+    rspec (3.5.0)
+      rspec-core (~> 3.5.0)
+      rspec-expectations (~> 3.5.0)
+      rspec-mocks (~> 3.5.0)
+    rspec-core (3.5.4)
+      rspec-support (~> 3.5.0)
+    rspec-expectations (3.5.0)
+      diff-lcs (>= 1.2.0, < 2.0)
+      rspec-support (~> 3.5.0)
+    rspec-mocks (3.5.0)
+      diff-lcs (>= 1.2.0, < 2.0)
+      rspec-support (~> 3.5.0)
+    rspec-support (3.5.0)
+    rubocop (0.48.1)
+      parser (>= 2.3.3.1, < 3.0)
+      powerpack (~> 0.1)
+      rainbow (>= 1.99.1, < 3.0)
+      ruby-progressbar (~> 1.7)
+      unicode-display_width (~> 1.0, >= 1.0.1)
+    ruby-hmac (0.4.0)
+    ruby-progressbar (1.9.0)
+    simple_oauth (0.3.1)
+    slop (3.6.0)
+    thor (0.20.0)
+    thread_safe (0.3.6)
+    timecop (0.7.1)
+    trollop (2.1.2)
+    twitter (5.17.0)
+      addressable (~> 2.3)
+      buftok (~> 0.2.0)
+      equalizer (= 0.0.10)
+      faraday (~> 0.9.0)
+      http (~> 1.0)
+      http_parser.rb (~> 0.6.0)
+      json (~> 1.8)
+      memoizable (~> 0.4.0)
+      naught (~> 1.0)
+      simple_oauth (~> 0.3.0)
+    unf (0.1.3)
+      unf_ext
+    unf_ext (0.0.7.4)
+    unicode-display_width (1.3.0)
+    xml-simple (1.1.5)
+    yard (0.8.7.6)
+
+PLATFORMS
+  ruby
+
+DEPENDENCIES
+  backup!
+  mocha (= 0.14.0)
+  pry
+  rake
+  rb-fsevent
+  rb-inotify
+  redcarpet
+  rspec (= 3.5.0)
+  rubocop (= 0.48.1)
+  timecop (= 0.7.1)
+  yard
+
+BUNDLED WITH
+   1.15.4
```

Comment:
This file should not be checked in. It's ignored in the `.gitignore` file. Did you add it manually?